### PR TITLE
SettingsWallet: readd change password button

### PR DESCRIPTION
**Pull request by selsta**
password

### CODE REVIEW

File: pages/settings/SettingsWallet.qml [hunk 1]
```diff
@@ -324,6 +324,84 @@ Rectangle {
...
+                onClicked: {
+                    passwordDialog.onAcceptedCallback = function() {
+	                    if(appWindow.walletPassword === passwordDialog.password){
+	                        newPasswordDialog.open()
+	                    } else {
+	                        informationPopup.title  = qsTr("Error") + translationManager.emptyString;
+	                        informationPopup.text = qsTr("Wrong password");
+	                        informationPopup.open()
+	                        informationPopup.onCloseCallback = function() {
+	                            changePasswordDialog.open()
```
Comment (mmbyday) [inline pages/settings/SettingsWallet.qml]:
```suggestion
```
qrc:/pages/settings/SettingsWallet.qml:394: ReferenceError: changePasswordDialog is not defined

Nicely done!

Comment (selsta) [reply pages/settings/SettingsWallet.qml]: Thanks :)

**Comment by xiphon** [comment]
I think it would be better to have a single `isHidden` flag controlling both the fields.

**Comment by selsta** [comment]
@xiphon updated

**Pull request merged**